shared/versions/stack/master.asciidoc

```diff
@@ -11,9 +11,9 @@ bare_version never includes -alpha or -beta
 :minor-version:          9.0
 :major-version:          9.x
 :prev-major-version:     8.x
-:prev-major-last:        8.17
+:prev-major-last:        8.18
 :major-version-only:     9
-:ecs_version:            8.17
+:ecs_version:            8.18
```

Comment:
Same here @lcawl , @bmorelli25 .

Also I have left master as 9.0 and only changed the references to 8.17 to 8.18. Let me know if this looks all right.

Reply:
IMO you shouldn't need to touch master.asciidoc at all.

Reply:
But `master` has this...

```
prev-major-last:        8.17
```

Shouldn't that line be bumped to 8.18?

Reply:
@lcawl , I'd need a response here. If you prefer I won't touch `master.asciidoc` at all, but the `prev-major-last` will be kept on 8.17.